### PR TITLE
[App Service] Fix #12653: az webapp log config --application-logging false doesn't turn it off

### LINKED ISSUE #12653: az webapp log config --application-logging false doesn't turn it off
# Issue
az webapp log config --application-logging false doesn't turn it off

## Default state
image

## Turn on
Run `az webapp log config --application-logging true -n mvpsdkdemoapp -o yaml` to turn on

```yaml
applicationLogs:
  azureBlobStorage:
    level: 'Off'
    retentionInDays: null
    sasUrl: null
  azureTableStorage:
    level: 'Off'
    sasUrl: null
  fileSystem:
    level: Error
detailedErrorMessages:
  enabled: false
failedRequestsTracing:
  enabled: false
httpLogs:
  azureBlobStorage:
    enabled: false
    retentionInDays: 7
    sasUrl: null
  fileSystem:
    enabled: false
    retentionInDays: 7
    retentionInMb: 100
id: /subscriptions/25fd0362-aa79-488b-b37b-d6e892009fdf/resourceGroups/mvpsdkdemorg/providers/Microsoft.Web/sites/mvpsdkdemoapp/config/logs
kind: null
location: West US
name: logs
resourceGroup: mvpsdkdemorg
type: Microsoft.Web/sites/config
```

## Turn off

To turn off I run the following:
`az webapp log config --application-logging false -n mvpsdkdemoapp -o yaml`

But the UI doesn't show it as disabled:

image

and the same output is returned.

```yaml
applicationLogs:
  azureBlobStorage:
    level: 'Off'
    retentionInDays: null
    sasUrl: null
  azureTableStorage:
    level: 'Off'
    sasUrl: null
  fileSystem:
    level: Error
detailedErrorMessages:
  enabled: false
failedRequestsTracing:
  enabled: false
httpLogs:
  azureBlobStorage:
    enabled: false
    retentionInDays: 7
    sasUrl: null
  fileSystem:
    enabled: false
    retentionInDays: 7
    retentionInMb: 100
id: /subscriptions/25fd0362-aa79-488b-b37b-d6e892009fdf/resourceGroups/mvpsdkdemorg/providers/Microsoft.Web/sites/mvpsdkdemoapp/config/logs
kind: null
location: West US
name: logs
resourceGroup: mvpsdkdemorg
type: Microsoft.Web/sites/config
```

> Please note that the applicationLogging: fileSystem only has "level", it doesn't have have "enabled"

# CLI version

```azure-cli                          2.2.0

command-modules-nspkg              2.0.3
core                               2.2.0
nspkg                              3.0.4
telemetry                          1.0.4

Python location 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe'     Extensions directory 'C:\Users\JonGallant\.azure\cliextensions'

Python (Windows) 3.6.6 (v3.6.6:4cf1f54eb7, Jun 27 2018, 02:47:15) [MSC v.1900 32 bit (Intel)]
```

azsdke2e

### PR DESCRIPTION
**Pull request by ThejaChoudary**
**Description
Fises #12653 
**  

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
webapp

File: azure-cli2017.pyproj [hunk 1]
```diff
@@ -13,8 +13,7 @@
     <LaunchProvider>Standard Python launcher</LaunchProvider>
     <InterpreterId>MSBuild|env|$(MSBuildProjectFullPath)</InterpreterId>
     <EnableNativeCodeDebugging>False</EnableNativeCodeDebugging>
-    <CommandLineArguments>
-    </CommandLineArguments>
+    <CommandLineArguments>webapp log config --name ps9085 --resource-group ps6148 --application-logging true</CommandLineArguments>
```
Comment (yungezz) [inline azure-cli2017.pyproj:16]: this should not be checked in to code

Comment (qwordy) [reply azure-cli2017.pyproj:16]: Furthermore, I doubt that we should put .pyproj file into git. It is a file designed for certain IDE.

File: src/azure-cli/azure/cli/command_modules/appservice/custom.py [hunk 2]
```diff
@@ -1950,10 +1950,10 @@ def config_diagnostics(cmd, resource_group_name, name, level=None,
 
     application_logs = None
     if application_logging is not None:
-        if not application_logging:
-            level = 'Off'
-        elif level is None:
-            level = 'Error'
+        if application_logging == 'true':
```
Comment (panchagnula) [inline src/azure-cli/azure/cli/command_modules/appservice/custom.py:1953]: @ThejaChoudary please add a test for this.

**Comment by yungezz** [comment]
hi @ThejaChoudary are you still on this PR? could you pls address comments and fix CI? thanks.

**Comment by ThejaChoudary** [comment]
Closing this as new [PR ](https://github.com/Azure/azure-cli/pull/14929)is raised.

**Pull request closed without merging**